### PR TITLE
Release 2021 02 02

### PR DESCRIPTION
**Pull request by kairu-ms**
## 2021.02.02 - Azure Ruby SDK Release
* Released the following SDKs
  - azure_mgmt_network - v0.26.0
  - azure_mgmt_compute - v0.21.0
  - azure_mgmt_operational_insights - v0.20.0
  - azure_mgmt_hdinsight - v0.18.5
  - azure_mgmt_cosmosdb - v0.23.0
  - azure_mgmt_confluent - v0.17.0
  - azure_sdk - v0.69.0

### CODE REVIEW

File: management/azure_mgmt_confluent/lib/profiles/latest/confluent_latest_profile_client.rb [hunk 1]
```diff
@@ -2,15 +2,15 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 
-require 'profiles/latest/compute_module_definition'
-require 'profiles/latest/modules/compute_profile_module'
+require 'profiles/latest/confluent_module_definition'
+require 'profiles/latest/modules/confluent_profile_module'
 
-module Azure::Compute::Profiles::Latest
+module Azure::Confluent::Profiles::Latest
```
Comment (agrare) [inline management/azure_mgmt_confluent/lib/profiles/latest/confluent_latest_profile_client.rb:8]: @kairu-ms was deleting this from `azure_mgmt_compute` intentional?  This appears to have made it into a released version of `azure_mgmt_compute` preventing it from being required.

https://github.com/Azure/azure-sdk-for-ruby/issues/2818

Comment (kairu-ms) [reply management/azure_mgmt_confluent/lib/profiles/latest/confluent_latest_profile_client.rb:8]: Thanks for your report. I will fix it soon.

Comment (agrare) [reply management/azure_mgmt_confluent/lib/profiles/latest/confluent_latest_profile_client.rb:8]: Unsure of your build/release pipeline process but I don't understand why you wouldn't fix it before merging this :shrug:

Comment (kairu-ms) [reply management/azure_mgmt_confluent/lib/profiles/latest/confluent_latest_profile_client.rb:8]: Sorry for that issue. It has been fixed in version 0.21.1. Please try again.

**Pull request merged**